tests/scalapack/scalapack_05.cc

```diff
@@ -62,7 +62,7 @@ void test(const unsigned int size, const unsigned int block_size)
   const NumberType rcond = scalapack_A.reciprocal_condition_number(scalapack_l1);
 
 
-  pcout << 1./(l1*inv_l1) << " " << rcond << std::endl;
+  pcout << 1./(l1*inv_l1) << " " << std::setprecision(7) << rcond << std::endl;
```

Comment:
from the output i noticed that `std::setprecision(7)` is being applied to both numbers from the second run. To make this consistent, could you please make it

```
pcout << std::setprecision(7) << 1./(l1*inv_l1) << " " << rcond << std::endl;
```

Reply:
(and adjust the output files)

Reply:
In fact, `pcout << std::setprecision(7)` modifies `pcout` not just for this statement, but for all following. Hence, I would just put it after the initialization of the `Utilities::MPI::MPI_InitFinalize` object.

Reply:
yes, that's even better.